src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -404,6 +421,35 @@ private MenuItem updateMarkAsReadUnreadMenuItem() {
         return markAsReadUnreadMenuItem;
     }
 
+    private MenuItem updateCloseReopenIssueMenuItem() {
+        Optional<GuiElement> item = listView.getSelectedItem();
+        if (item.isPresent()) {
+            closeReopenIssueMenuItem.setDisable(false);
+            TurboIssue selectedIssue = item.get().getIssue();
+
+            if (selectedIssue.isOpen()) {
+                closeReopenIssueMenuItem.setText(closeIssueMenuItemText);
+            } else {
+                closeReopenIssueMenuItem.setText(reopenIssueMenuItemText);
+            }
+        } else {
+            closeReopenIssueMenuItem.setDisable(true);
```

Comment:
This `else` clause should also come first, and you can also use `return` to reduce nesting `if/else`